README.md

```diff
@@ -4,6 +4,8 @@
 
 The Ansible Community AWS collection includes a variety of Ansible content to help automate the management of AWS instances. This collection is maintained by the Ansible community.
 
+AWS related Modules and plugins supported by the Ansible Cloud team are in the [amazon.aws](https://github.com/ansible-collections/amazon.aws) repository.
```

Comment:
```suggestion
AWS related modules and plugins supported by the Ansible Cloud team are in the [amazon.aws](https://github.com/ansible-collections/amazon.aws) collections.
```

Reply:
@jillr thank you. I've used singular - collection. - same as in [amazon.aws README](https://github.com/ansible-collections/community.aws/pull/258). Hope it's ok. Or is there any particular reason for plural?

Reply:
That was a typo on my part - thanks!